src/components/Footer.tsx

```diff
@@ -0,0 +1,33 @@
+import { TodoFilter } from './TodoFilter';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  setFilterValue: (filterValue: string) => void;
+  filterValue: string;
+  completedTodos: Todo[];
+  activeTodos: Todo[];
+};
+
+export const Footer: React.FC <Props> = ({
+  setFilterValue, filterValue, completedTodos, activeTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {activeTodos.length}
+        {' '}
+        items left
```

Comment:
Combine it into one template string.

[More info.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
npm run lint change it

Reply:
{`${activeTodos.length} items left`}

Reply:
not fixed